packages/server-ct/src/routes-ct.ts

```diff
@@ -1,12 +1,22 @@
-const httpProxy = require('http-proxy')
-const send = require('send')
-const debug = require('debug')('cypress:server:routes')
-
+import httpProxy from 'http-proxy'
+import send from 'send'
+import { Express, ErrorRequestHandler } from 'express'
+import debugFn from 'debug'
+import Project from './project-ct'
 // const files = require('@packages/server/lib/controllers/files')
-const runnerCt = require('@packages/runner-ct')
-const staticPkg = require('@packages/static')
+import runnerCt from '@packages/runner-ct'
+import staticPkg from '@packages/static'
+
+const debug = debugFn('cypress:server:routes')
```

Comment:
did you check that the debugs are appearing when asked? I have never seen the default of debug.

Reply:
This always works for me